HW_9/HW_9_Task_2.js

```diff
@@ -0,0 +1,53 @@
+const characters = [
+    { 'name': 'Barney', 'age': 36 },
+    { 'name': 'Fred', 'age': 40 },
+    { 'name': 'Jack', 'age': 50 }
+  ];
+
+
+
+// 1. Напишите функцию addCharacter(character) позволяющую добавить новый объект в массив characters. 
+//     Объект должен иметь поля name (string) и age (number)
+function addCharacter(character) {characters.push(character);}
+
+addCharacter({'name':'Boris', 'age':46});
+
+console.log(characters);
+
+
+// 2. Напишите функцию getCharacter(name), позволяющую получить объект персонажа по его имени// getCharacter('Fred') => { 'name': 'Fred', 'age': 40 }
+function getCharacter(name) {
+    return character = characters.find((element) => element.name === name);
```

Comment:
+ Желательно обработку для случая, если не нашли персонажа
```suggestion
    return characters.find((element) => element.name === name);
```